docs-chef-io/content/inspec/resources/opa_api.md

```diff
@@ -0,0 +1,69 @@
++++
+title = "opa_api resource"
+draft = false
+gh_repo = "inspec"
+platform = "os"
+
+[menu]
+  [menu.inspec]
+    title = "opa_api"
+    identifier = "inspec/resources/os/opa_api.md mongodb_conf resource"
```

Comment:
should this be mongodb_conf @IanMadd ?

Reply:
Thanks, @tas50. I have created PR fot that.